ale/spiceql_access.py

```diff
@@ -18,6 +16,12 @@
 if log_level is not None and log_level.lower() == "debug":
     logger.setLevel(logging.DEBUG)
 
+try:
+    import pyspiceql
+except ImportError:
+    logger.info("Optional package 'pyspiceql' was not imported, " \
+                "install 'spiceql' to use this feature. " \
+                "NOTE: SpiceQL is not supported on Windows.") 
```

Comment:
This would cause errors whenever you import, did you test this locally to make sure this doesn't cause issues?